win32ss/user/user32/misc/dllmain.c

```diff
@@ -544,16 +544,12 @@ DllMain(
 
             if (!gfServerProcess)
             {
-#if WIN32K_ISNT_BROKEN
```

Comment:
So does it correctly work now? (This hack was in order to fix [CORE-10540](https://jira.reactos.org/browse/CORE-10540))
In the corresponding commit message you can reference that it's a removed hack from commit https://github.com/reactos/reactos/commit/1b33a3c6aae04f4bcf9d9bf954f089699a17e32c

Reply:
I don't know yet.

Reply:
@ThFabba Can you check about this hack removal?

Reply:
As long as `InitializeImmEntryTable` somehow takes a reference on user32, you should be fine to remove this. You can run `user32_dynamic_apitest` and see if it regresses

Reply:
BEFORE THIS PR:
![before-pr](https://user-images.githubusercontent.com/2107452/130014026-476c71d7-e966-430d-b3c5-57bef6c2f06f.png)
BEFORE REMOVAL:
![before-removal](https://user-images.githubusercontent.com/2107452/130013310-c9dc9f3d-41da-4353-96d9-b3a0dd272dc6.png)
AFTER REMOVAL:
![after-removal](https://user-images.githubusercontent.com/2107452/130013349-25fbd3f6-bcd8-4e61-871b-2f7a2d864c70.png)
There was no change.